### ISSUE TITLE
Signed covenant but then button doesnt load

### ISSUE THREAD

**Issue by GriffGreen** [1Hive/honey-pot#262]
<img width="799" alt="Screen Shot 2021-03-07 at 9 33 56 AM" src="https://user-images.githubusercontent.com/14945613/110243603-65b45280-7f5b-11eb-9e36-8c6c91c692bd.png">

Yay!!!

But ummmmm... now what?
<img width="799" alt="Screen Shot 2021-03-07 at 9 33 56 AM" src="https://user-images.githubusercontent.com/14945613/110243615-75339b80-7f5b-11eb-8f4e-f56a204bb621.png">

**Comment by GriffGreen** [comment]
I think i want to be sent back to the home page with the new button... and it would be awesome if i had some confetti after i signed the covenant and joined the community

**Comment by rperez89** [comment]
You mean that after succesfully signed the agreement you suggest to take to the home screen?

**Comment by GriffGreen** [comment]
@rperez89 yes, exactly.... take me to where I want to go... or at least give me a pop up that tells me how awesome i am for taking that action and then has a button that takes me there
